src/AgentContainer.cpp

```diff
@@ -251,8 +249,6 @@ void AgentContainer::moveAgentsToHome ()
     }
 
     m_at_work = false;
-
-    Redistribute();
```

Comment:
@stevenhofmeyr specifically added the `Redistribute()` in [this PR](https://github.com/AMReX-Codes/ExaEpi/pull/79). Can they really be removed?